### PR TITLE
ECOSYSTEM: add the Svix webhooks service

### PR DESCRIPTION
**Pull request by tasn**

### CODE REVIEW

File: ECOSYSTEM.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ If your project isn't listed here and you would like it to be, please feel free
 - [realworld-axum-sqlx](https://github.com/launchbadge/realworld-axum-sqlx): A Rust implementation of the [Realworld] demo app spec using Axum and [SQLx].
 - [Rustapi](https://github.com/ndelvalle/rustapi): RESTful API template using MongoDB
 - [Jotsy](https://github.com/ohsayan/jotsy): Self-hosted notes app powered by Skytable, Axum and Tokio
+- [Svix](https://www.svix.com) ([repository](https://github.com/svix/svix-webhooks)): The enterprise ready webhook service
```
Comment (jplatte) [inline ECOSYSTEM.md:25]:
```suggestion
- [Svix](https://www.svix.com) ([repository](https://github.com/svix/svix-webhooks)): Enterprise-ready webhook service
```

**Comment by tasn** [comment]
Updated, thanks! I was just using our tagline... :)

**Comment by davidpdrsn** [comment]
Thanks!

**Comment by jplatte** [comment]
> Updated, thanks! I was just using our tagline... :)

Then that is also missing the hyphen 😛

**Pull request merged**